package Problema2;
//Problema 2
//Controla la excepción del ejercicio anterior, y da un aviso al usuario de “número incorrecto” si
//sucede un error.

public class Principal {

    public static void main(String[] args) throws Exception {

        Texto texto = new Texto();
        texto.deTextoANumero();

    }


}
